src/typescript.ts

```diff
@@ -21,7 +21,7 @@ import * as ts from 'typescript';
ion, ExportSpecifier, Expression, ExpressionStatement, ExpressionWithTypeArguments, flattenDiagnosticMessageText, forEachChild, formatDiagnostics, FormatDiagnosticsHost, FunctionDeclaration, FunctionLikeDeclaration, GetAccessorDeclaration, getCombinedModifierFlags, getCombinedNodeFlags, getLeadingCommentRanges, getLineAndCharacterOfPosition, getMutableClone, getOriginalNode, getPreEmitDiagnostics, getSyntheticLeadingComments, getSyntheticTrailingComments, getTrailingCommentRanges, HeritageClause, Identifier, IfStatement, ImportDeclaration, ImportEqualsDeclaration, ImportSpecifier, IndexKind, InterfaceDeclaration, isArrowFunction, isAssertionExpression, isBinaryExpression, isCallExpression, isExportAssignment, isExportDeclaration, isExpressionStatement, isExternalModule, isExternalModuleReference, isFunctionDeclaration, isIdentifier, isImportDeclaration, isImportEqualsDeclaration, isInterfaceDeclaration, isLiteralExpression, isLiteralTypeNode, isMethodSignature, isModuleBlock, isModuleDeclaration, isNamedImports, isNamespaceExportDeclaration, isNamespaceImport, isNonNullExpression, isObjectLiteralExpression, isParenthesizedExpression, isPropertyAccessExpression, isPropertyAssignment, isPropertyDeclaration, isPropertySignature, isQualifiedName, isSourceFile, isStringLiteral, isTypeReferenceNode, isVariableDeclaration, isVariableStatement, MethodDeclaration, Modifier, ModifierFlags, ModuleBlock, ModuleDeclaration, ModuleKind, ModuleResolutionHost, NamedDeclaration, NamedImports, NamespaceExportDeclaration, Node, NodeArray, NodeBuilderFlags, NodeFlags, NonNullExpression, NotEmittedStatement, ObjectFlags, ObjectLiteralElementLike, ObjectLiteralExpression, ObjectType, ParameterDeclaration, parseCommandLine, parseJsonConfigFileContent, Program, PropertyAccessExpression, PropertyAssignment, PropertyDeclaration, PropertyName, PropertySignature, QualifiedName, readConfigFile, resolveModuleName, ScriptTarget, SetAccessorDeclaration, setCommentRange, setEmitFlags, setOriginalNode, setSourceMapRange, setSyntheticLeadingComments, setSyntheticTrailingComments, setTextRange, Signature, SignatureDeclaration, SignatureKind, SourceFile, Statement, StringLiteral, Symbol, SymbolFlags, SymbolFormatFlags, SyntaxKind, SynthesizedComment, sys, TextRange, ThisExpression, Token, TransformationContext, Transformer, TransformerFactory, Type, TypeAliasDeclaration, TypeAssertion, TypeChecker, TypeElement, TypeFlags, TypeNode, TypeParameterDeclaration, TypeReference, TypeReferenceNode, UnionType, updateAsExpression, updateBlock, updateConstructor, updateExportDeclaration, updateExpressionWithTypeArguments, updateGetAccessor, updateHeritageClause, updateMethod, updateNamedExports, updateNonNullExpression, updateParameter, updatePropertyAccess, updateSetAccessor, updateSourceFileNode, updateTypeAssertion, VariableDeclaration, VariableStatement, visitEachChild, visitFunctionBody, visitLexicalEnvironment, visitNode, Visitor, visitParameterList, WriteFileCallback} from 'typescript';
```

Comment:
Note, had to pass through the `Bundle` type here since it appears in the `ts.CustomTransformers` type